### ISSUE TITLE
Make python-fints compatible with Python 3.10

### ISSUE THREAD

**Issue by jamilraichouni** [raphaelm/python-fints#134]
**Describe the bug**
`python-fints` is not compatible with Python3.10

![image](https://user-images.githubusercontent.com/45113557/143115315-d7198adf-e049-43bf-9efa-b0f2622ccd6a.png)

**Bank I tested this with**
None, see above

**Expected behavior**
`python-fints` is compatible with Python3.10

**Code required to reproduce**
```python
import fints.client.FinTS3PinTanClient
```

**Log output / error message**
```python
>>> import fints.client.FinTS3PinTanClient
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jamilraichouni/.venvs/finances-44HoV-h9-py3.10/lib/python3.10/site-packages/fints/client.py", line 14, in <module>
    from .connection import FinTSHTTPSConnection
  File "/Users/jamilraichouni/.venvs/finances-44HoV-h9-py3.10/lib/python3.10/site-packages/fints/connection.py", line 9, in <module>
    from .message import FinTSInstituteMessage, FinTSMessage
  File "/Users/jamilraichouni/.venvs/finances-44HoV-h9-py3.10/lib/python3.10/site-packages/fints/message.py", line 3, in <module>
    from .formals import SegmentSequence
  File "/Users/jamilraichouni/.venvs/finances-44HoV-h9-py3.10/lib/python3.10/site-packages/fints/formals.py", line 3, in <module>
    from fints.fields import *
  File "/Users/jamilraichouni/.venvs/finances-44HoV-h9-py3.10/lib/python3.10/site-packages/fints/fields.py", line 6, in <module>
    from fints.types import Container, SegmentSequence, TypedField
  File "/Users/jamilraichouni/.venvs/finances-44HoV-h9-py3.10/lib/python3.10/site-packages/fints/types.py", line 1, in <module>
    from collections import Iterable, OrderedDict
ImportError: cannot import name 'Iterable' from 'collections' (/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/collections/__init__.py)
```
**Additional context**
N/A

**Comment by lukasjuhrich** [comment]
This has already been adressed in  #132 (901805f1c0a0a4785db0abba7ff770fb97e095a2), however, a new version has not been released yet. A workaround is replacing any `fints~=[version]` line in your requirements file with `git+git://github.com/raphaelm/python-fints.git@846c25f5f6ae578cde5151fc30b5f7a3601a95ac#egg=fints`

**Comment by clstaudt** [comment]
Although the workaround does its job, it would be great to have a new release including Python 3.10 support.

**Comment by raphaelm** [comment]
Sorry 'bout the delay, released 3.1.0 just now!
